src/Catty/Store/StoreProjectReporter.swift

```diff
@@ -0,0 +1,108 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+import Foundation
+
+protocol StoreProjectReporterProtocol {
+    func report(projectId: String, message: String, completion: @escaping (StoreProjectReporterError?) -> Void)
+}
+
+final class StoreProjectReporter: StoreProjectReporterProtocol {
+
+    private let httpBoundary = "---------------------------98551263596598746108249098092---------------------------"
+    private let contentTypeTag = "Content-Type"
+    private let contentLengthTag = "Content-Length"
+    private let authorizationTag = "Authorization"
+    private let standardCategory = "Inappropriate"
+    let session: URLSession
+
+    init(session: URLSession = StoreProjectReporter.defaultSession()) {
+        self.session = session
+    }
+
+    func report(projectId: String, message: String, completion: @escaping (StoreProjectReporterError?) -> Void) {
+
+        let urlString = NetworkDefines.reportProjectUrl
+        var request = URLRequest.init(url: URL(string: urlString)!)
+
+        guard let postData = String(format: "program=%@&note=%@&category=%@", projectId, message, standardCategory).data(using: .utf8, allowLossyConversion: true) else { return }
+        let contentType = "multipart/form-data; boundary=\(httpBoundary)"
+
+        if let token = JNKeychain.loadValue(forKey: NetworkDefines.kUserLoginToken) as? String {
+            let tokenString = String(format: "Bearer %@", token)
```

Comment:
Please remove the "Bearer " prefix and send the token alone.

Reply:
Removed!